google-beta/resource_project_service_identity_test.go

```diff
@@ -0,0 +1,39 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccProjectServiceIdentity_basic(t *testing.T) {
+	t.Parallel()
+
+	resourceName := "google_project_service_identity.hc_sa"
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckHealthcareDatasetDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testGoogleProjectServiceIdentity_basic(),
+			},
+			{
```

Comment:
You can delete this step, I don't think it does anything. Normally we'd import the resource, but obviously this doesn't support that.

Reply:
Done.